source/features/quick-comment-edit.tsx

```diff
@@ -7,6 +7,33 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import isArchivedRepo from '../helpers/is-archived-repo';
 
+function addQuickEditButton(commentForm: Element): void {
+	commentForm.classList.add('rgh-edit-comment');
+
+	const comment = commentForm.closest('.js-comment')!;
+	if (select.exists('.rgh-quick-comment-edit-button', comment)) { // #5572
```

Comment:
But we already have a deduplication code: `.js-comment-update:not(.rgh-edit-comment)`

If that doesn't work, either we move the class higher in the tree or we drop it